services/debugger/cache/cache.go

```diff
@@ -0,0 +1,46 @@
+package cache
+
+//go:generate mockgen -destination=../../../mocks/services/debugger/cache/mock_cache.go -package=mocks_cache github.com/rudderlabs/rudder-server/services/debugger/cache CacheAny
+
+import (
+	"github.com/rudderlabs/rudder-server/services/debugger/cache/embeddedcache"
+	"github.com/rudderlabs/rudder-server/services/debugger/cache/listcache"
+	"github.com/rudderlabs/rudder-server/utils/logger"
+)
+
+var pkgLogger logger.Logger
```

Comment:
Can we avoid global variables please? The logger can simply be passed as a dependency ([example](https://github.com/rudderlabs/rudder-server/blob/v1.4.5/enterprise/suppress-user/repository.go#L34)).

Reply:
Yeah will refactor accordingly by trying not using global variables.